crates/pop-cli/src/commands/build/contract.rs

```diff
@@ -19,6 +19,12 @@ impl BuildContractCommand {
 	pub(crate) fn execute(&self) -> anyhow::Result<()> {
 		clear_screen()?;
 		intro(format!("{}: Building a contract", style(" Pop CLI ").black().on_magenta()))?;
+
+		tokio::spawn(pop_telemetry::record_cli_command(
```

Comment:
Could you add imports for `tokio::spawn`, `pop_telemetry::record_cli_command` and `serde_json::json` in main.rs so that all usages can be simplified to a one liner such as `spawn(record_cli_command("build", json!({"contract": ""})));`

Actually, would it not be much easier to push this up the stack to https://github.com/r0gue-io/pop-cli/blob/353161d8b03936f1ca1e36fbb41cd9d942b6f649/crates/pop-cli/src/main.rs#L47 so that all the telemetry is defined in a single place - i.e. in the match arm before call command.execute()?